core/src/main/java/org/jdbi/v3/core/qualifier/QualifiedType.java

```diff
@@ -57,13 +64,21 @@ public static QualifiedType of(Type type, Annotation qualifier) {
 
     /**
      * Creates a QualifiedType for {@code type} with the given qualifiers.
+     *
+     * @param type the type
+     * @param qualifiers the qualifiers
+     * @return the QualifiedType
```

Comment:
Probably a source of merge conflicts here. We changed to `of(type).with(qualifiers...)` style